pages/dao/[symbol]/proposal/components/instructions/Dual/StakingOption.tsx

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import React, { useContext, useEffect, useState } from 'react'
+import {
+  UiInstruction,
+  DualFinanceStakingOptionForm,
+} from '@utils/uiTypes/proposalCreationTypes'
+import { NewProposalContext } from '../../../new'
+import GovernedAccountSelect from '../../GovernedAccountSelect'
+import { Governance } from '@solana/spl-governance'
+import useGovernanceAssets from '@hooks/useGovernanceAssets'
+import { ProgramAccount } from '@solana/spl-governance'
+import Input from '@components/inputs/Input'
+import getConfigInstruction from '@utils/instructions/Dual'
+import useWalletStore from 'stores/useWalletStore'
+import { getDualFinanceStakingOptionSchema } from '@utils/validations'
+import Tooltip from '@components/Tooltip'
```

Comment:
nit, deduplicate `@solana/spl-governance` import: 

```suggestion
import React, { useContext, useEffect, useState } from 'react'
import { ProgramAccount, Governance } from '@solana/spl-governance'
import {
  UiInstruction,
  DualFinanceStakingOptionForm,
} from '@utils/uiTypes/proposalCreationTypes'
import { NewProposalContext } from '../../../new'
import GovernedAccountSelect from '../../GovernedAccountSelect'
import useGovernanceAssets from '@hooks/useGovernanceAssets'
import Input from '@components/inputs/Input'
import getConfigInstruction from '@utils/instructions/Dual'
import useWalletStore from 'stores/useWalletStore'
import { getDualFinanceStakingOptionSchema } from '@utils/validations'
import Tooltip from '@components/Tooltip'
```